report_qweb_element_page_visibility/views/layouts.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
```

Comment:
v9 gave us a nifty `<odoo>` to use instead of this. You can also delete the `<data>` tag now.